kpi/deployment_backends/kobocat_backend.py

```diff
@@ -115,30 +119,39 @@ def internal_to_external_url(url):
 
     @property
     def backend_response(self):
-        return self.asset._deployment_data['backend_response']
+        return self.get_data('backend_response', {})
 
     def _kobocat_request(self, method, url, **kwargs):
         """
         Make a POST or PATCH request and return parsed JSON. Keyword arguments,
         e.g. `data` and `files`, are passed through to `requests.request()`.
+
+        `kwargs` contains arguments to be passed to KoBoCAT request, but it can
+        also contain `expect_formid` which bypasses presence of 'formid'
+        property in KoBoCAT response.
         """
 
         expected_status_codes = {
+            'GET': 200,
             'POST': 201,
             'PATCH': 200,
             'DELETE': 204,
         }
+
         try:
             expected_status_code = expected_status_codes[method]
         except KeyError:
             raise NotImplementedError(
                 'This backend does not implement the {} method'.format(method)
             )
 
+        expect_formid = kwargs.pop('expect_formid', True)
```

Comment:
what's the advantage of this over including it in the signature? personally, i'd prefer the self-documenting clarity of naming the argument in the signature if possible.
```
In [1]: def f(a, b, **kwargs): 
   ...:     print(a, b, kwargs.get('dog'), kwargs.get('cat')) 
   ...:                                                                                                                              

In [2]: f('a', 'b', cat='clyde', dog='rocky')                                                                                        
a b rocky clyde

In [3]: def f(a, b, dog='cotton', **kwargs): 
   ...:     print(a, b, dog, kwargs.get('cat')) 
   ...:                                                                                                                              

In [4]: f('a', 'b', cat='clyde', dog='rocky')                                                                                        
a b rocky clyde
```

Reply:
No advantage, let's add it to the signature.

Reply:
✅